spec/spec_helper.rb

```diff
@@ -33,10 +33,10 @@ def image_path(path, options = {})
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
 
 RSpec.configure do |config|
-  config.expect_with :rspec do |c|
+  config.expect_with(:rspec) do |c|
```

Comment:
Confused at why we have changes to the `spec/` dir, when you ignore it in https://github.com/ManageIQ/guides/pull/228.